Jenkinsfiles/utils.groovy

```diff
@@ -18,11 +19,29 @@ def get_target() {
     if (env.BRANCH_NAME == STAGE_BRANCH_NAME) {
         return 'stage'
     }
+    if (env.BRANCH_NAME == STAGE_BRANCH_NAME) {
```

Comment:
Hm, I just noticed that we're using `get_target()` in two different ways: as the target setup script name, like `prod` or `prod.mm` (see `make()` function) and as the target system name like `prod`, `stage`, or `standby` (see `announce_push()` function).

Perhaps we should do something like?
  - change the name of `get_target()` to `get_target_script()`
  - call `get_target_script()` instead of `get_target()` in the `make()` function
  - create a new `get_target_system()` function, with the same code as `get_target_script()` except that it returns `'standby'` for the `STANDBY_BRANCH_NAME` case
  - call `get_target_system()` instead of `get_target()` in the `announce_push()` function

Reply:
Good suggestion. I'm using ``get_target_name``, to mirror ``get_repo_name``, instead of ``get_target_system``.